en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md

```diff
@@ -17,11 +17,11 @@ API Manager product. Following is a list of key features supported by Choreo Con
 - Scope validation for JWT OAuth2 bearer tokens.
 - Applying CORS configuration from publisher portal.
 - Support full lifecycle states of APIs like created, prototyped, blocked, deprecated, published and etc.
-- Support virtual hosts(Vhosts) to expose the APIs.  
+- Support virtual hosts (Vhosts) to expose the APIs.  
 - API/Resource, application and subscription level throttling.
 - Advance throttling, custom policies and the blocking conditions.
 - Publish analytics to the Choreo cloud.
-- Back end JWT generation (Passing end user details to the back end services).
+- Back-end JWT generation (Passing end user details to the back end services).
```

Comment:
backend is a one word.

Reply:
Corrected with: https://github.com/wso2/docs-apim/pull/4961/commits/ee1a0f528455b6e24a2ba1d2458f83fc4c2e740c